.release-please-manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-  ".": "1.9.2",
-  "packages/api-explorer": "0.9.12",
-  "packages/code-editor": "0.1.5",
-  "packages/extension-api-explorer": "21.8.2",
-  "packages/extension-sdk": "21.8.3",
-  "packages/extension-sdk-react": "21.8.3",
-  "packages/hackathon": "21.8.2",
-  "packages/run-it": "0.9.12",
-  "packages/sdk": "21.8.3",
-  "packages/sdk-codegen": "21.0.20",
-  "packages/sdk-codegen-scripts": "21.0.21",
+  ".": "1.10.0",
+  "packages/api-explorer": "0.9.13",
+  "packages/code-editor": "0.1.6",
+  "packages/extension-api-explorer": "21.10.0",
+  "packages/extension-sdk": "21.10.0",
+  "packages/extension-sdk-react": "21.10.0",
+  "packages/hackathon": "21.10.0",
+  "packages/run-it": "0.9.13",
+  "packages/sdk": "21.10.0",
+  "packages/sdk-codegen": "21.0.21",
+  "packages/sdk-codegen-scripts": "21.0.22",
   "packages/sdk-codegen-utils": "21.0.11",
-  "packages/sdk-node": "21.8.2",
-  "packages/sdk-rtl": "21.0.16",
-  "packages/wholly-sheet": "0.5.11",
-  "python": "21.8.1"
+  "packages/sdk-node": "21.10.0",
+  "packages/sdk-rtl": "21.0.17",
+  "packages/wholly-sheet": "0.5.12",
+  "python": "21.10.0"
```

Comment:
everything that should have bumped from `21.8.x` to `21.10.0` now has [WTF](https://github.com/googleapis/release-please/pull/958#issuecomment-876603190)

Reply:
My take-away is we're still good to go here?